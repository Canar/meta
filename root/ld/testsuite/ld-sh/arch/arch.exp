../../../../../binutils-gdb/ld/testsuite/ld-sh/arch/arch.exp