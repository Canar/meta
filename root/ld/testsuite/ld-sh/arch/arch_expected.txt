../../../../../binutils-gdb/ld/testsuite/ld-sh/arch/arch_expected.txt